src/components/Grid/SearchComponent.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState }from 'react';
+import {tableData} from './TableData';
+
+
+const CLASS_NAMES = {
+    SEARCH_MAIN_CONTAINER: 'search-main-container'
+}
+
+const SearchComponent = (props) => {
+
+const [groupList, setGroupList] = useState(tableData);
+const [filteredGroupNameOrId, setfilteredGroupNameOrId] = useState([]);
+
+    const handleChange = (e) => {
+        let currentGroups = [];
+        e.preventDefault();
+        console.log("value", e.target.value)
+        // let filteredGroups = [];
+
+        if (e.target.value !== '') {
+            currentGroups = groupList;
+            filteredGroupNameOrId = currentGroups.filter(name => {
+                const lowercase = name.toLowerCase();
+                const filter = e.target.value.toLowerCase();
```

Comment:
performance trick - do the `e.target.value.toLowerCase();` outside the `filter`

Reply:
fixed